### PR TITLE
Add fields descriptions for other settings pages

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds fields descriptions to the:
- Configurations page
- Product Reviews
- Schedule Design Update

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/product-scheduled-design-update.html
https://docs.magento.com/user-guide/catalog/settings-advanced-product-reviews.html
https://docs.magento.com/user-guide/catalog/product-configurations.html

### CODE REVIEW

File: src/catalog/product-configurations.md [hunk 1]
```diff
@@ -9,3 +9,17 @@ _Configurations Section_
 
 ![]({% link images/images/product-configurations-hoodie.png %}){: .zoom}
 _Product Configurations_
+
+## Field reference
+
+|Field|Description|
+|--- |--- |
+|Image|Product image.|
```
Comment (dobooth) [inline src/catalog/product-configurations.md:17]: Generally, we don't use periods in these table descriptions, unless it is an actual sentence.

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
And there is a merge conflict here.

**Comment by ZhovnirMaria** [comment]
@dobooth Thanks, done ✅ 
Have a good day 😊

File: src/catalog/settings-advanced-product-reviews.md [hunk 2]
```diff
@@ -7,6 +7,22 @@ The _Product Reviews_ section lists all reviews that customers have submitted ab
...
+|Field|Description|
+|--- |--- |
+|ID|Unique, numeric ID generated for the product review entry|
+|Created|Date of publication of the review|
+|Status|Review status. Options: **Pending**, **Approved**, **Not Approved**.|
+|Title|Review title|
+|Nickname|The nickname of the user who left the review|
+|Review|Customer review on the current product|
+|Visibility|Visibility in store reviews|
+|Type|Type of user who left the review. Options: **Guest**, **Customer**.|
```
Comment (leslietilling) [inline src/catalog/settings-advanced-product-reviews.md:21]:
```suggestion
|Type|Type of user who left the review (Guest or Customer)|
```

File: src/catalog/settings-advanced-product-reviews.md [hunk 3]
```diff
@@ -7,6 +7,22 @@ The _Product Reviews_ section lists all reviews that customers have submitted ab
 ![]({% link images/images/product-review.png %}){: .zoom}
 _Product Reviews_
 
+## Field reference
+
+|Field|Description|
+|--- |--- |
+|ID|Unique, numeric ID generated for the product review entry|
+|Created|Date of publication of the review|
+|Status|Review status. Options: **Pending**, **Approved**, **Not Approved**.|
```
Comment (leslietilling) [inline src/catalog/settings-advanced-product-reviews.md:16]:
```suggestion
|Status|Review status (Pending, Approved, or Not Approved)|
```

File: src/catalog/settings-advanced-product-reviews.md [hunk 4]
```diff
@@ -7,6 +7,22 @@ The _Product Reviews_ section lists all reviews that customers have submitted ab
...
+|Review|Customer review on the current product|
+|Visibility|Visibility in store reviews|
+|Type|Type of user who left the review. Options: **Guest**, **Customer**.|
+|Product|Reviewed product name|
+|SKU|The unique Stock Keeping Unit that is assigned to the product|
+|Action|Opens the product in Edit mode|
+
 ## Moderation of product reviews
 
 1. On the _Admin_ sidebar, go to **Catalog** > **Products** > Edit the Product > _Product Reviews_ section.
```
Comment (leslietilling) [inline src/catalog/settings-advanced-product-reviews.md:28]: Let's fix this while we are here

```suggestion
1. On the _Admin_ sidebar, go to **Catalog** > **Products**.

1. Locate the product and open it in edit mode.

1. Scroll to the _Product Reviews_ section.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
A few minor edits for style consistency

**Comment by ZhovnirMaria** [comment]
@leslietilling, hi!
Thank you, Done ☑️

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**